src/main/java/com/spring/boot/project/demo/service/GenericService.java

```diff
@@ -0,0 +1,21 @@
+package com.spring.boot.project.demo.service;
+
+import java.util.List;
+
+public interface GenericService<T,I> {
```

Comment:
```suggestion
public interface GenericService<T,ID> {
```
Так буде більш читабельно

Reply:
Да согласен, опирался на то что Generics обозначают одной большой букваой.

Reply:
Оставил так как есть из-за того что не проходит тест на code style.